config/vendor-api-v1.yml

```
@@ -419,7 +419,7 @@ components:
       properties:
         support_reference:
           type: string
-          description: The candidate’s reference number for their application in the Apply system
+          description: The candidate’s reference number for their application form carrying this application choice in the Apply system. We show it to candidates when they submit their form. Candidates can have multiple application forms. For instance, when a candidate moves from Apply 1 to Apply again their candidate ID will stay the same, but that’s a new application form so the `support_reference` will be different.
```

```suggestion
          description: The candidate’s reference number for their application form carrying this application choice in the Apply system. Candidates can have multiple application forms. For instance, when a candidate moves from Apply 1 to Apply again their candidate ID will stay the same, but that’s a new application form so the `support_reference` will be different.
```

We no longer show on submission